### PR TITLE
Include help document for the Bootstrap Carousel

### PR DESCRIPTION
**Pull request by lukos**
I have used this widget in my project and thought I might as well add a guide document. Of course, you can edit/move the files and content if you think it would work better some other way.

### CODE REVIEW

File: docs/guide/usage-carousel.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  In this example, the controls do not use the default scroll icons, which are just the < and > arrows, but instead use the
+  glyphicons that are used on the bootstrap example on their site.
+  
+  showIndicators is whether you want the small "radio" buttons that can click through onto each slide. This is true by default.
+  
+  The items then have one required field (that is the content for the slide) and the second field, which is an optional caption.
+  
+  As with most Yii widgets, there are options arrays which allow you to further customise the displayed widget.
+
+## Using holder.js ##
```
Comment (samdark) [inline docs/guide/usage-carousel.md]: It's not part of the official bootstrap and has quite specific use-case. Don't think it should be mentioned.

**Comment by samdark** [comment]
Thanks for the effort but it seems to be tutorial on specific use-case rather than a general purpose guide.

**Comment by lukos** [comment]
I think it matches the widgets you supply in this package. The idea would be to have examples for each of the controls otherwise everyone has to work it out by themselves.

**Comment by samdark** [comment]
Well, `holder.js` is not part of the official bootstrap and is only one of possibly hundreds various solutions for the same topic. The topic itself isn't general enough to be included.

**Comment by samdark** [comment]
The rest is duplicating phpdoc pretty much.

**Comment by lukos** [comment]
OK. Fair enough. I might see if any of this could be moved into the PHP doc instead.

**Comment by samdark** [comment]
It's probably OK to add an example for each widget but it should not use third party stuff or suggest how to do not so common things such as these placeholders.

**Pull request closed without merging**